README.md

```diff
@@ -56,6 +56,20 @@ Visit the [releases page](https://github.com/1dot13/source/releases) to download
    (those issues can occur due to the combination of old game and modern OS/hardware, cnc-ddraw helps to avoid those) 
 
 
+### First Time Building on Windows
+
+1. Run `Visual Studio 2019` or newer.
+2. Click `Clone a repository`, enter `https://github.com/1dot13/source.git` on `Location` and click `Clone`.
+3. Double-click on `Folder View` on the `Solution Explorer`
+4. You will get a CMake error saying `No existing preset was found, copied a preset template to [some_path]`. This is normal and only happens once.
+5. Click on the dropdown that says `x64-Debug` and select `Manage configurations...`. This should trigger Visual Studio to load the `CMakeUserPresets.json` file it just copied.
+6. Click `Build -> Build All` to build an English Ja2 v.1.13 Debug executable.
+
+To configure other languages and executables such as Unfinished Business and Map Editor, click the configuration dropdown (where it says `1dot13 Debug`, `1dot13 RelWithDebInfo` or `1dot13 Release`), choose `Manage Configurations...` and edit the `"Languages"` and `"Applications"` field to suit your needs.
+
+For debugging within the IDE, put the path to the Ja2 1.13 installation (with appropriate `gamedir` installation!) on the `CMAKE_RUNTIME_OUTPUT_DIRECTORY` field.
+
+
```

Comment:
```suggestion
Visual Studio setup
Run Visual Studio 2019 or newer.
Clone and open the location with the source code using one of these two options:
Click Clone a repository
Enter git@github.com:1dot13/source.git or https://github.com/1dot13/source.git in the Repository location field, select the path you want to clone the repository to and click Clone.
Double-click on Folder View in the Solution Explorer
Click Open a local folder
Use this option if you already cloned the repository yourself.
Visual Studio will automatically detect the CMake configuration files and will run the CMake generation. There will bet a CMake error in the logs saying No existing preset was found, copied a preset template to [some_path]. This is normal and only happens once.
Click on the dropdown that says x64-Debug and select Manage configurations.... This should trigger Visual Studio to load the CMakeUserPresets.json file it just copied. Now you can close the window for managing the configurations.
The x64-Debug target should have been replaced by 1dot13 Debug. Click it and select Manage configurations... again. Here is where you configure the language for the built executables as well as which ones to build, Most important, here is where you set CMAKE_RUNTIME_OUTPUT_DIRECTORY to the path to your JA2 1.13 installation. This will be used for debugging. Note that the path needs to have a working 1.13 installation, and that includes the 1.13 game data.
You can use Build -> Build All to build the executables you selected in the configuration.


```